src/circuit/operator/apply.rs

```diff
@@ -37,3 +37,31 @@ where
         (self.func)(i1)
     }
 }
+
```

Comment:
Why do we need this operator when we can just write `stream.apply(|x| x.to_string())`?